stable/openvpn/templates/config-openvpn.yaml

```diff
@@ -82,14 +75,16 @@ data:
       key-direction 1
       {{- end }}
       EOF
-      cat pki/$1.ovpn
 
   revokeClientCert.sh: |-
       #!/bin/bash
+      set -e
+
       EASY_RSA_LOC="/etc/openvpn/certs"
       cd $EASY_RSA_LOC
-      ./easyrsa revoke $1 
-      ./easyrsa gen-crl
+      export EASYRSA_PKI=$EASY_RSA_LOC/pki
+      easyrsa revoke $1
+      easyrsa gen-crl
```

Comment:
Is there a reason for not using `./` before `easyrsa` anymore? maybe it's now in the path in the new image, but I think we either `export EASYRSA_PKI` or `cd $EASY_RSA_LOC` consistently as I don't believe the `cd` is needed anymore if we're exporting

Reply:
It is in the path in the new image, yeah. I’ll try and clean up the cd-ing.